snippets/dig.md

```diff
@@ -0,0 +1,56 @@
+### dig
+
+Return the value in a nested JSON object by the key.
+
+Given the key name (or target), it will look up the key in the object recursively and return the first value if found.
+
+```
+const dig = (obj, target) => {
+  if (!obj) return;
+
+  if (obj[target]) return obj[target];
```

Comment:
I'm pretty sure, if you remove the error check above, you can wrap this whole thing in a one-liner with the ternary operator (`?`) or two and avoid the curly brackets around the function as well as the `return` statement.